repos/system_upgrade/el7toel8/actors/checkmemory/libraries/library.py

```diff
@@ -0,0 +1,44 @@
+from leapp.models import MemoryInfo
+from leapp.libraries.common.config import architecture
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.stdlib import api
+from leapp.reporting import create_report
+from leapp import reporting
```

Comment:
This was the issue. You are importing the reporting library and the create_report function as well, but you are mocking just the function inside the reporting library. Not the one imported in the tested library.

Reply:
@fellipeh that's the part of the code I have been talking about.

Reply:
Ok @pirat89 ! Now I understand! :)   Thanks!